src/makerom/rom_header.s

```diff
@@ -1,18 +1,15 @@
-/*
- * The Legend of Zelda: Ocarina of Time ROM header
- */
+#include "rom_header.h"
 
-.byte  0x80, 0x37, 0x12, 0x40   /* PI BSD Domain 1 register */
-.word  0x0000000F               /* Clockrate setting */
-.word  0x80000400               /* Entrypoint function (`entrypoint`) */
-.word  0x0000144C               /* Revision */
-.word  0x917D18F6               /* Checksum 1 */
-.word  0x69BC5453               /* Checksum 2 */
-.word  0x00000000               /* Unknown */
-.word  0x00000000               /* Unknown */
-.ascii "THE LEGEND OF ZELDA "   /* Internal ROM name */
-.word  0x00000000               /* Unknown */
-.word  0x0000004E               /* Cartridge */
-.ascii "ZL"                     /* Cartridge ID */
-.ascii "P"                      /* Region */
-.byte  0x0F                     /* Version */
+ENDIAN_IDENTIFIER
+PI_DOMAIN_1_CFG(64, 18, 7, 3)
+SYSTEM_CLOCK_RATE_SETTING(0xF)
+ENTRYPOINT(0x80000400)
+LIBULTRA_VERSION(2,0,L)
+CHECKSUM(0)
+PADDING(8)
+ROM_NAME("THE LEGEND OF ZELDA")
+PADDING(7)
+MEDIUM(CARTRIDGE)
+GAME_ID("ZL")
+REGION(PAL)
+GAME_REVISION(15)
```

Comment:
is it worth adding offset comments to this file? I've had to count out the bytes here more than once when staring at a rom hexdump, but now it's a bit harder to do that